pkgs/os-specific/linux/lenovo-legion/app.nix

```diff
@@ -17,6 +17,7 @@ python3.pkgs.buildPythonApplication rec {
 
   propagatedBuildInputs = with python3.pkgs; [
     pyqt5
+    pyqt6
```

Comment:
Do we need both 5 and 6?

Reply:
yeah seems like its dependent on both, removing it yields the following error

```
error: builder for '/nix/store/1zqwzwgyihajzqad8m50kawmd5zd2m2y-lenovo-legion-app-0.0.12.drv' failed with exit code 1;
       last 10 log lines:
       >
       >         ********************************************************************************
       >         Requirements should be satisfied by a PEP 517 installer.
       >         If you are using pip, you can try `pip install --use-pep517`.
       >         ********************************************************************************
       >
       > !!
       >   ir_d = dist.fetch_build_eggs(dist.install_requires)
       > /nix/store/gd3shnza1i50zn8zs04fa729ribr88m9-python3-3.11.8/bin/python3.11: No module named pip
       > error: Command '['/nix/store/gd3shnza1i50zn8zs04fa729ribr88m9-python3-3.11.8/bin/python3.11', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--
no-deps', '-w', '/build/tmpkbr2av9s', '--quiet', 'PyQT5']' returned non-zero exit status 1.
       For full logs, run 'nix log /nix/store/1zqwzwgyihajzqad8m50kawmd5zd2m2y-lenovo-legion-app-0.0.12.drv'.
```

Reply:
hmmm it could also be due to the libsforqt5.qtbase.. let me see if that changes anything

Reply:
yeah nope, can't get the pkg to compile with qt6.qtbase owing to a similar error.

Reply:
Okay, no problem :)